src/meta/test/MetaClientTest.cpp

```diff
@@ -468,9 +469,18 @@ TEST(MetaClientTest, HeartbeatTest) {
     auto listener = std::make_unique<TestListener>();
     auto clientPort = network::NetworkUtils::getAvailablePort();
     HostAddr localHost{localIp, clientPort};
+    std::string clusterIdPath = "/tmp/storage.cluster.id.";
```

Comment:
We don't need to read cluster id from file in UTs

Reply:
Fixed!